README.md

```diff
@@ -111,6 +112,20 @@ mergeImages(['./body.png', './eyes.png', './mouth.png'], {
   // data:image/png;base64,iVBORw0KGgoAA...
 ```
 
+**node-canvas 2.x**
+```js
+const mergeImages = require('merge-images');
+const { Canvas, Image } = require('canvas');
+
+Canvas.Image = Image;
```

Comment:
Mutating something you don't own feels bad to me, which is why I opened https://github.com/lukechilds/merge-images/pull/81

Reply:
Makes sense, I just wanted to avoid any potential breaking changes from using canvas 1.x with it